lib/BloqadeGates/Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "BloqadeGates"
 uuid = "9b473568-af80-491c-8f51-e251334462c1"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
why change the version number without any updates in the code?

Reply:
The codes changed in the last PR #472 but I forgot to update the version number.